### PR TITLE
[1.18] Add logic for running openshift e2e-aws tests

### PR DESCRIPTION
**Pull request by umohnani8**
Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

File: contrib/test/ci/cri-o.spec [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+%define gobuild(o:) scl enable go-toolset-1.%{gominver} -- go build -buildmode pie -compiler gc -tags="rpm_crashtraceback no_openssl ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags'" -a -v -x %{?**};
+%else
+%define gobuild(o:) go build -buildmode pie -compiler gc -tags="rpm_crashtraceback no_openssl ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags'" -a -v -x %{?**};
+%endif
+
+%global provider github
+%global provider_tld com
+%global project cri-o
+%global repo cri-o
+# https://github.com/kubernetes-sigs/cri-o
```
Comment (haircommander) [inline contrib/test/ci/cri-o.spec:26]: nit, it's been a long time since we lived in kubernetes-sigs :)

Comment (umohnani8) [reply contrib/test/ci/cri-o.spec]: haha good catch :) fixed

**Pull request merged**